### ISSUE TITLE
maybe overfitting?

### ISSUE THREAD

**Issue by ranjiewwen** [joelthchao/tensorflow-finetune-flickr-style#13]
- thanks for you code. now i am run the code . 
- but i fond maybe the model overfitting.

```
2019-05-21 16:26:02.627367 Iter 5040: Training Loss = 0.0148, Accuracy = 1.0000
2019-05-21 16:26:07.453140 Iter 5060: Training Loss = 0.0249, Accuracy = 1.0000
2019-05-21 16:26:12.326071 Iter 5080: Training Loss = 0.0172, Accuracy = 1.0000
2019-05-21 16:26:17.227455 Iter 5100: Training Loss = 0.0552, Accuracy = 0.9800
2019-05-21 16:26:27.450521 Iter 5120: Testing Accuracy = 0.4336
```
- there we can see , train loss is lower and train accuracy is near 1.0 ;while the test accuracy is  0.4336 .
- later i will make some change to make test accuracy more high. I don't know if my understanding is right 。

**Comment by joelthchao** [comment]
Yes, it's obviously overfitting according to your log. You can put some regularization to prevent it.
